docs/vault/installation.md

```diff
@@ -56,17 +57,21 @@ If the Vault spends funds from another wallet this may be marked as theft.
 (Optional) If you already have a locally running Bitcoin testnet node, only start the Vault client:
 
 ```shell
-docker-compose up vault
+docker-compose up vault -d
 ```
 
 ?> You may need to edit the docker-compose to point `--bitcoin-rpc-url` to `http://localhost:18332`.
 
 You can run the entire Vault client and the Bitcoin node with the following command:
 
 ```shell
-docker-compose up
+docker-compose up -d
 ```
 
+You can optionally view the running docker containers with command `docker ps`.
```

Comment:
Not 100% sure, but shouldn't docker-compose be sufficient? `docker-compose ps`, `docker-compose logs interbtc-vault_vault_1`, .. from the folder should work.

Reply:
You can use both, docker and docker-compose for this task
I have however changed it to using docker-compose, if that's the one you like better.
The output of each one are slightly different and also the command changes:

With docker:
```
docker ps
docker logs interbtc-vault_vault_1 -f
docker logs interbtc-vault_bitcoind_1 -f
```

With docker-compose:
```
docker-compose ps
docker-compose logs -f vault
docker-compose logs -f bitcoind
```